corehq/apps/locations/forms.py

```diff
@@ -334,11 +334,17 @@ def save(self, metadata):
         return location
 
 
+class LocationUserForm(NewMobileWorkerForm):
+    def clean_location_id(self):
+        # This is handled separately
```

Comment:
Can we include the reason why we are doing this a little in detail? That would help understand the need of this change quickly and better when someone sees it.

Reply:
updated